src/index.js

```diff
@@ -11,65 +11,93 @@ import StackedColumnWidget from './components/stackedColumnWidget/stackedColumnW
+            {/*<path d="M 0 0 L 5 10 L 10 0" stroke="currentColor" strokeWidth="2"></path>*/}
+          {/*</pattern>*/}
         </defs>
       </svg>
     </div>
 
-    <SparklineWidget chartConfig={{"xAxis":{
-        "categories":["Jul","Aug","Sep","Oct","Nov","Dec","Jan"]
-      },
-      "series":[
-        {"name":"Total opportunities","data":[null,null,13,29,42,58,74]}
-      ]
-    }} title="Total opportunities"
-                     units="number"
-                     type="sparkline"
-                     dateLastUpdated="2017-02-01T23:11:18.675Z"
-                     _singleCategory={false}
-                     _singleSection={true}
-                     minimumValue="13" />
+
+            {/*<SparklineWidget chartConfig={{"xAxis":{*/}
+        {/*"categories":["Jul","Aug","Sep","Oct","Nov","Dec","Jan"]*/}
+      {/*},*/}
+      {/*"series":[*/}
+        {/*{"name":"Total opportunities","data":[null,null,13,29,42,58,74]}*/}
+      {/*]*/}
+    {/*}} title="Total opportunities"*/}
+                     {/*units="number"*/}
+                     {/*type="sparkline"*/}
+                     {/*dateLastUpdated="2017-02-01T23:11:18.675Z"*/}
+                     {/*_singleCategory={false}*/}
+                     {/*_singleSection={true}*/}
+                     {/*minimumValue="13" />*/}
 
     <DonutWidget chartConfig={{"series":[{
-        name: "Jan",
-        data: [
-          {"name":"Mobile","y":183},
-          {"name":"Tablet","y":30},
-          {"name":"Desktop","y":200}
-        ]
-      }]
-    }} title="Devices used"
-                 units="percentage"
-                 type="donut"
-                 dateLastUpdated="2017-02-01T01:02:02.240Z"
-                 _singleCategory={true}
-                 _singleSection={false}
-                 minimumValue="30" />
+          name: "Jan",
+          data: [
+            {"name":"Mobile","y":183},
+            {"name":"Tablet","y":30},
+            {"name":"Desktop","y":200}
+          ]
+        }]
+      }} title="Devices used"
+       units="percentage"
+       type="donut"
+       dateLastUpdated="2017-02-01T01:02:02.240Z"
+       _singleCategory={true}
+       _singleSection={false}
+       minimumValue="30"
+       displayHighContrast={true} />
 
-    <StackedColumnWidget
-      chartConfig={{
-        "xAxis":{"categories":["May","Jun","Jul","Aug","Sep","Oct","Nov","Dec","Jan"]},
-        "series":[
-          {"name":"Public Beta","data":[0,0,2,2,2,2,2,2,2]},
-          {"name":"Pre Beta","data":[0,0,1,3,3,3,3,3,3]},
-          {"name":"Non-transformational","data":[1,1,1,1,1,1,1,1,1]},
-          {"name":"Other","data":[0,0,0,0,0,0,0,0,0]}
-        ]
-      }}
-      title="Types of service"
-      units="number"
-      type="stackedColumn"
-      dateLastUpdated="2017-02-01T01:02:02.240Z"
-      _singleCategory={false}
-      _singleSection={false}
-      stackingType="normal"
-    />
+    {/*<StackedColumnWidget*/}
```

Comment:
it's basically a scratch pad this file, im happy to keep the sample code commented out, useful when working on StackedColumnWidget